docs/PROJECT_CONSTITUTION.md

```
@@ -0,0 +1,1410 @@
+# MonsterBorg Self-Drive Project Constitution
+
+**Version**: 1.0
+**Effective Date**: 2025-11-14
+**Project**: MonsterBorg Self-Drive System
+**Repository**: https://github.com/dxcSithLord/monster-self-drive
```

markdownlint-cli2 (0.18.1)

Bare URL used

(MD034, no-bare-urls)